suite/tests/linux/clone.c

```diff
@@ -47,7 +47,7 @@
 #include <string.h>
```

Comment:
(Discussing offline the plan for handling complications with importing clone.c changes internally.)